guides/common/modules/proc_creating-a-webhook.adoc

```diff
@@ -0,0 +1,55 @@
+[id="creating-a-webhook_{context}"]
+= Creating a Webhook
+
+You can customize events, payloads, HTTP authentication, content type and headers through {ProjectWebUI}. 
+Use the following procedure to create a webhook in the {ProjectWebUI}.
+
+.Procedure
+
+. In the {ProjectWebUI}, click *Administer* -> *Webhooks*.
+. Click *Create Webhook*.
+. Click *Subscribe to field* to select an event.
+. Enter a name.
+. Enter a target URL.
+  When using the shellhooks plugin, the URL should be in the form of `https://{smartproxy.example.com}:{smartproxy_port}/shellhooks/my_script`.
```

Comment:
```suggestion
  When using the shellhooks plugin, the URL should be in the form of `https://{smartproxy-example-com}:{smartproxy_port}/shellhooks/my_script`.
```
This wasn't implemented from the previous review.

Reply:
Ok, I see that now. I fixed that in two places. Thanks!